js/src/common/utils/classList.ts

```diff
@@ -7,10 +7,10 @@
  * // "foo qux"
  *
  * @param {Object} classes
- * @return {String}
+ * @return {string}
  */
-export default function classList(classes) {
-  let classNames;
+export default function classList(classes: Object): string {
```

Comment:
This can be replaced with the following (not tested, off the top of my head):
```ts
function classList(classes: { [ name: string]: boolean }) {
```